corpus/accounts/migrations/0001_initial.py

```diff
@@ -71,7 +71,7 @@ class Migration(migrations.Migration):
                 (
                     "date_joined",
                     models.DateTimeField(
-                        default=django.utils.timezone.now, verbose_name="date joined"
+                        default=django.utils.timezone.localtime, verbose_name="date joined"
```

Comment:
Don't change the migration file, change the `models.py` file and then generate a new migration.

Reply:
Fixed in new commit